tsl/src/compression/compression.c

```diff
@@ -1323,6 +1327,11 @@ row_compressor_flush(RowCompressor *row_compressor, CommandId mycid, bool change
 		row_compressor->compressed_values[compressed_col] = 0;
 		row_compressor->compressed_is_null[compressed_col] = true;
 	}
+
+	if (NULL != row_compressor->on_flush)
+		row_compressor->on_flush(row_compressor,
+								 row_compressor->rows_compressed_into_current_value);
```

Comment:
I assume the for following PRs we'll have some tests for covering it, right?

Reply:
yes, there is no code coverage now because nothing is using it. But the table access method will be using it.